app/views/kalibro_configurations/new.html.erb

```diff
@@ -1,3 +1,3 @@
-<h1>New Configuration</h1>
+<h1><%= "#{t('new.female')} #{KalibroConfiguration.model_name.human}" %></h1>
```

Comment:
We are going to discuss this soon. I do agree with you but fixing this will be a history of its own.